_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,79 @@
+
+## OpenSearch
+
+See the following example `YAML` configuration file for the `otel_trace_group` processor when you connect to an OpenSearch cluster using your username and password:
+
+``` YAML
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://localhost:9200"]
+        cert: path/to/cert
+        username: YOUR_USERNAME_HERE
+        password: YOUR_PASSWORD_HERE
+```
+
+See [OpenSearch security]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sinks/opensearch/#amazon-opensearch-service-domain-security) for a more detailed explanation of which OpenSearch credentials and permissions are required and how to configure those credentials for the OTel trace group processor.
+
+### Amazon OpenSearch Service
+
+See the following example `YAML` configuration file for the `otel_trace_group` processor when you use Amazon OpenSearch Service:
+
+``` YAML
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://your-amazon-opensearch-service-endpoint"]
+        aws_sigv4: true
+        cert: path/to/cert
+        insecure: false
+```
+
+## Configuration
+
+You can configure the `otel_trace_group` processor with the following options.
+
+| Name | Description | Default value |
+| -----| ----| -----------|
+| `hosts`| A list of IP addresses of OpenSearch nodes. Required. | No default value. | 
+| `cert` | A certificate authority (CA) certificate that is PEM-encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that OpenSearch is using. | `null` |
+| `aws_sigv4` | A Boolean flag to sign the HTTP request with AWS credentials. Only applies to Amazon OpenSearch Service. See [OpenSearch security](https://github.com/opensearch-project/data-prepper/blob/129524227779ee35a327c27c3098d550d7256df1/data-prepper-plugins/opensearch/security.md) for details. | `false`. |
+| `aws_region` | A string that represents the Region of Amazon OpenSearch Service domain, for example, `us-west-2`. Only applies to Amazon OpenSearch Service. | `us-east-1` |
+| `aws_sts_role_arn`| An Identity and Access Management (IAM) role that the sink plugin assumes to sign the request to Amazon OpenSearch Service. If not provided, the plugin uses the [Default credentials](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/auth/credentials/DefaultCredentialsProvider.html). | `null` |
+| `aws_sts_header_overrides` | A map of header overrides to make when assuming the IAM role for the sink plugin. | `null` |
+| `insecure` | A Boolean flag to turn off SSL certificate verification. If set to `true`, CA certificate verification is turned off and insecure HTTP requests are sent. | `false` |
+| `username` | A string that contains the username and is used in the [internal users](https://opensearch.org/docs/latest/security/access-control/users-roles/) of your OpenSearch cluster. | `null` |
```

Comment:
Here and in the next line, "used in the internal users" is unclear. Can we clarify?